docs/source/topics/cli.rst

```diff
@@ -127,18 +127,6 @@ is identical to
     verdi --verbosity debug process list
 
 When the option is specified multiple times, only the last value will be considered.
-The default value for the logging level can be changed permanently through the configuration system.
```

Comment:
for the daemon logs this is still mentioned somewhere else, right?

Reply:
Yep: https://aiida.readthedocs.io/projects/aiida-core/en/latest/intro/troubleshooting.html?highlight=logging.aiida_loglevel#increasing-the-logging-verbosity